ekf/ekflib.c

```diff
@@ -70,12 +73,41 @@ int ekf_init(void)
 	/* TODO: config read should utilize parser, and default values should be stored in /etc/calib.conf */
 	kmn_configRead(&ekf_common.initVals); /* only for development process */
 
+	/* TODO: reimplement this nasty cast below */
+	ekf_common.initVals.log = (int)(*((float*)(&ekf_common.initVals.log)));
+
+	/* Choose ekf log mode */
+	switch (ekf_common.initVals.log) {
+		case 1:
+			logFlags = EKFLOG_SENSC;
+			break;
+		case 2:
+			logFlags = EKFLOG_MEAS;
+			break;
+		case 3:
+			logFlags = EKFLOG_EKF_IMU;
+			break;
+		case 4:
+			logFlags = EKFLOG_EKF_POS;
+			break;
+		default:
+			logFlags = 0;
+	}
+
+	if (ekflog_init("logfile.txt", logFlags) != 0) {
```

Comment:
The best way would be to put the date into the log name but currently it would not work for zynq-7000.
```suggestion
	if (ekflog_init("ekf_log.txt", logFlags) != 0) {
```